content/applications/productivity/spreadsheet/filters.rst

```diff
@@ -0,0 +1,205 @@
+=======
+Filters
+=======
+
+A filter is a tool that presents specific data in a spreadsheet within the *Documents* app.
+
+Different filter options are available to use, depending on the data contained in the spreadsheet.
+If a specific filter does not apply to a spreadsheet, that filter is not available.
+
+.. example::
+   A spreadsheet contains only information about the quantities of product in a warehouse, and does
+   **not** have any date-related information. No filters relating to dates appear in the presented
+   filters list.
+
+Filters work by first setting the parameters for a filter, then making selections *within those
+parameters* to present specific data. Each filter is created in this two-part process: setting
+parameters, then specifying the values of the parameters.
+
+To access the available filters for a spreadsheet, first navigate to the :menuselection:`Documents
+app`, then click on the desired spreadsheet. Next, click the :icon:`fa-filter` :guilabel:`(Filters)`
+icon in the top-right corner to view the available filters.
+
+.. image:: filters/filter.png
+   :align: center
+   :alt: The filter icon in the top-right corner of a spreadsheet.
+
+Add a new filter
+================
+
+If no filters have been configured for a spreadsheet, a new filter must first be added.
+
+Click the :icon:`fa-filter` :guilabel:`(Filters)` icon in the top-right corner, and a
+:guilabel:`Filters` popover window appears.
+
+Beneath the heading :guilabel:`Add a new filter...`, click on one of the presented filter options:
+:ref:`Date <filters/date>`, :ref:`Relation <filters/relation>`, or :ref:`Test <filters/text>`. The
+options presented are the only filters that apply to the selected spreadsheet.
+
+.. image:: filters/add-filter.png
+   :align: center
+   :alt: The filter options that appear after clicking the filter icon.
+
+Below are all the various options that can appear in the :guilabel:`Add a new filter...` list.
```

Comment:
Should add an anchor link here, or something else to make it clear that the options are below the following paragraph. It just feels a bit broken up